docs/Developer's ReadMe.md

```diff
@@ -0,0 +1,209 @@
+
+2. There are two places to localize strings: in the package.json file and in the typescript files in the src directory.
+
+   - If you want to add a new string to the package.json file, replace the string with your key enclosed by the percent sign as such \% **key** \% i.e. `"This is a string"` becomes `"%exampleProperty.exDescription%"`. Then go to the package.nls.json file found in the root directory of the repository and include your newly created key and string inside as a json key/value pair.
+
+   - If you want to add a new string to a typescript file, you will need to include the following library in your file (if not already included). `import * as nls from 'vscode-nls';`
+
+     - You will also need to include the following code:
+
+       ```Typescript
+       // Set up localization
+       nls.config({ messageFormat: nls.MessageFormat.bundle, bundleFormat: nls.BundleFormat.standalone })();
+       const localize: nls.LocalizeFunc = nls.loadMessageBundle();
+       ```
+
+     - Next wrap your string with the localize function in this format `localize('key', 'string') i.e. localize('addEntry','Successfully called add entry.')`
+
+3. After adding/updating/removing any string, run `yarn run package`. This will update the sample directory under the i18n folder with the newly added strings. Upload these files to Zanata or email a maintainer to do so.
+
+Maintainers: Evann Wu (evannw@andrew.cmu.edu), Lauren Li (lauren.li@ibm.com), Kristina Mayo (ktopchi@us.ibm.com)
+
+### Adding a New Language
+
+1. Navigate to the i18n folder found in the root directory of the repository. Duplicate the sample folder and rename the new folder to the ISO-639-3 code for the language [found here](<https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Languages/List_of_ISO_639-3_language_codes_(2019)>).
+
+2. Once the language has been translated, go to the Zowe VS Code Extension project in Zanata, select the most up to date version, select the translated language, and for each file, press the arrow to the left of the filename and select download translated .json.
+
+3. Replace the files in the folder you created with these newly downloaded files of the same name.
+
+4. Next, open gulpfile.js found in the root directory of the repository. Add the following information: { folderName: `'ISO-639-3-Code'`, id: `'vscode-locale-id'` } to the `languages` array. For example, for Chinese add: `{ folderName: 'zho', id: 'zh-cn' }`. You can find the vscode locale id [here](https://code.visualstudio.com/docs/getstarted/locales).
+
+5. Make sure you have the vscode language pack of this new language installed and to see the localized result, first run the `yarn run package` command in terminal. Then press F1, run the Configure Display Language command, and select the locale id of your translated language.
```

Comment:
same here: `yarn workspace vscode-extension-for-zowe package` or just refer to first section